modules/openapi-generator/src/main/resources/dart/libraries/dio/analysis_options.mustache

```diff
@@ -3,7 +3,7 @@ analyzer:
     strict-inference: true
     strict-raw-types: true
   strong-mode:
-    implicit-dynamic: false
+    implicit-dynamic: true 
```

Comment:
no sure that's wanted, why changing this ? if it's a requirement for the lib please do it only for it

Reply:
Sorry this wasn't meant to be committed. The code generated by `json_serializable` includes a lot of implicit dynamic and so this option throws up a lot of errors in the generated code.

How do we feel about excluding `*.g.dart` instead, so only the generated code is excluded? Building on that, only exclude generated code if we're using json_serializable?

Reply:
to me doesn't make sens to analyze generated code, so I'm fine to exclude `*.g.dart` for all

Reply:
I am not sure you can exclude files from `strong-mode`. I thought this only works for the lints.

Reply:
This seems to work? https://github.com/google/json_serializable.dart/issues/557#issuecomment-618260593